### DISCUSSION TITLE
%player_exp_to_level% not working??

### DISCUSSION THREAD

**Discussion by TinyMandalorian** [PlaceholderAPI/PlaceholderAPI#700] [(2) Support] [answered]
![image](https://user-images.githubusercontent.com/67829574/131127699-5d69c8bb-6862-4a51-854c-22efe663ced0.png)

is %player_exp_to_level% broken cause the levels seem to be completely off and i got no idea why!

**Comment by Andre601** [comment, marked as answer]
To my knowledge does `%player_exp_to_level%` tell you how many XP you need to reach the next level.

**Reply by TinyMandalorian** [reply]
oh ok then what can i use to show there level

**Reply by Andre601** [reply]
`%player_level%`

**Reply by TinyMandalorian** [reply]
thx
